recipes/markitdown/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "markitdown" %}
-{% set version = "0.0.1a2" %}
+{% set version = "0.0.1dev2" %}
```

Comment:
You have to follow CFEP-05 or ask upstream for a stable release.

Reply:
I changed this back to `0.0.1a2`, but now override the channel target - would that be an acceptable path forwards?

Reply:
You only need to remember to remove the conda `conda_build_config.yaml` file when updating this to a stable release.

Reply:
Makes sense 👍